packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -13,21 +12,26 @@ type DomainSuggestion = import('@automattic/data-stores').DomainSuggestions.Doma
 /**
  * Internal dependencies
  */
-import { STORE_KEY } from '../../stores/onboard';
-import { FLOW_ID } from '../../constants';
-import { RecordTrainTracksEventProps } from '../../lib/analytics';
-import { getSignupDomainsSuggestionsVendor } from '../../utils/domain-suggestions';
+// import { getSuggestionsVendor } from 'lib/domains/suggestions';
 
-const DOMAIN_SUGGESTION_VENDOR = getSignupDomainsSuggestionsVendor();
+// const DOMAIN_SUGGESTION_VENDOR = getSuggestionsVendor( true );
+
+// TODO: Fix this.
+const DOMAIN_SUGGESTION_VENDOR = 'variation4_front';
```

Comment:
😱 Need to fix this.

Reply:
In a call we talked to;
- pull it from the `lib/domains/suggestions` in Gutenboarding
- pass it as a prop to the package

Reply:
Done.